templates/focus-ios.xliff

```diff
@@ -28,6 +28,19 @@
       </trans-unit>
     </body>
   </file>
+  <file original="Blockzilla/en.lproj/Intents.strings" source-language="en" datatype="plaintext">
+    <header>
+      <tool tool-id="com.apple.dt.xcode" tool-name="Xcode" tool-version="9.1" build-num="9B55"/>
+    </header>
+    <body>
+      <trans-unit id="Erase">
+        <source>Erase</source>
+      </trans-unit>
+      <trans-unit id="Erase browsing session">
```

Comment:
this doesn't look like a valid ID (should probably be more something that looks like "Erase.Browsing.Session")